lib/cli/scripts/init-aae-env.ts

```diff
@@ -93,6 +186,14 @@ async function deployMissingApps(args: ConfigArgs) {
     }
 }
 
+async function healthCheckMicroServices(args: ConfigArgs, alfrescoJsApi: any, result: any) {
+    await healthCheckDeployment(args, alfrescoJsApi, result);
+    await healthCheckModeling(args, alfrescoJsApi, result);
+    await healthCheckModeling(args, alfrescoJsApi, result);
```

Comment:
Do we need to check the Modeling service twice?

Reply:
ops